Dockerfile-2.6

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:trusty
+
+RUN echo "deb http://archive.ubuntu.com/ubuntu trusty main universe" > /etc/apt/sources.list && \
+	echo "deb http://archive.ubuntu.com/ubuntu trusty-updates main universe" >>/etc/apt/sources.list && \
+	echo "deb http://archive.ubuntu.com/ubuntu trusty-security main universe" >>/etc/apt/sources.list && \
+	apt-get update && \
+	apt-get install -qq -y openssl software-properties-common && \
+	add-apt-repository ppa:fkrull/deadsnakes && \
+	apt-get update && \
+	apt-get install -qq -y python2.6 python2.6-dev wget make && \
+	(cd /tmp && wget https://bootstrap.pypa.io/get-pip.py && python2.6 get-pip.py && rm get-pip.py) && \
+	apt-get clean && rm -rf /var/cache/apt/archives/* /var/lib/apt/lists/*
+
+COPY requirements.txt /opt/project/requirements.txt
+
+WORKDIR /opt/project
+
+RUN grep -v '^-e .' requirements.txt >/tmp/requirements.txt && \
+	pip install -r /tmp/requirements.txt && \
+	rm /tmp/requirements.txt
+
+COPY . /opt/project
+
+RUN pip install -e .
```

Comment:
I'm filtering out "-e ." in first step of dependencies install - so we don't need to reinstall them each time

This way rebuild after sources change is mostly cached